devops/pipeline/e2etest.yaml

```diff
@@ -1,6 +1,9 @@
 # E2E test pipeline - performs end-to-end tests on the following platforms
+#  - Debian 9
 #  - Ubuntu 18.04
 #  - Ubuntu 20.04
+# 
+# See "!!! ADD NEW DISTROS HERE !!!" for new distro placeholder
```

Comment:
>See "!!! ADD NEW DISTROS HERE !!!" for new distro placeholder [](http://example.com/codeflow?start=2&length=61)

How about: "Add new distros here as needed:"

Reply:
No lots of !s, no all uppercase...

Reply:
This comment here does not appear necessary. The others in the body can switch to be normal, as suggested above.